diagnostics/README.md

```diff
@@ -1,4 +1,4 @@
-# kafka-daq-report
+# diagnostics
 
 A simple TUI tool that listens on the trace topic and reports in a table view the following for each DAQ that is seen to be sending messages:
```

Comment:
This documentation looks like it needs to be updated to reflect the new modes available.